package.json

```diff
@@ -19,6 +19,7 @@
     "test:coverage": "npm run test:coverage -w packages",
     "test:examples": "npm run test -w examples --if-present",
     "watch": "tsc --build --watch packages/core packages/hardhat-plugin",
+    "typedocs": "typedoc --entryPointStrategy packages --out ./typedocs ./packages/core",
```

Comment:
What's this?

Reply:
That is a documentation generator that reads the typescript source. I am using it to quickly visualise the types in the package and their documentation.

I suspect it will not make it into the final version of the PR once it has done its job.